HexCtrl/src/Helper.h

```diff
@@ -39,6 +39,9 @@ namespace HEXCTRL {
 		//Converts every two numeric chars to one respective hex character: "56"->V(0x56), "7A"->z(0x7A)
 		bool StrToHex(const std::string& strFrom, std::string& strToHex);
 
+		//Convert string to GUID
+		bool StringToGuid(const wchar_t* pwszSource, LPGUID pGUIDResult);

```

Comment:
`Helper.h` possesses only project wide stuff.
What is this `CHexDlgDataInterpret` related function doing here?

Reply:
I was trying to be helpful because it could be useful later. However, I will move it now as you wish.